model/Listener/ReplaceCopiedQtiXmlIdentifierListener.php

```
@@ -44,7 +45,13 @@ class ReplaceCopiedQtiXmlIdentifierListener
      */
     public static function catchItemCreatedFromSource(ItemContentClonedEvent $itemContentClonedEvent): void
     {
-        ServiceManager::getServiceManager()
+        $serviceManager = ServiceManager::getServiceManager();
```

PHPMD (2.15.0)

Avoid using static access to class '\oat\oatbox\service\ServiceManager' in method 'catchItemCreatedFromSource'. (Clean Code Rules)

(StaticAccess)